src/Controller/UserController.php

```diff
@@ -3,22 +3,29 @@
 namespace App\Controller;
 
 use App\Entity\Users;
+use App\Repository\UsersRepository;
+use App\Response\UserResponse;
 use DateTime;
 use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\HttpFoundation\JsonResponse;
+use App\Transformer\SimpleTransformer;
+use App\Validator\UserRequestValidator;
+use Codeception\Util\HttpCode;
+use Exception;
 use Phpass\Hash;
 
 class UserController extends AbstractController
 {
     private $em;
-    
+    private $usersRepository;
```

Comment:
Please add type hints fot those fields and rename $em to proper name eg: `$entityManager`